tests/browser_context_test.go

```diff
@@ -0,0 +1,180 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/dop251/goja"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBrowserContextAddCookies(t *testing.T) {
```

Comment:
👏 This is great! Just one comment more: could you make these tests run in parallel? Take a look at [this](https://github.com/zucchinho/xk6-browser/blob/2aa3ad11b531c0613e39d8a6a70f3d7120605bb2/tests/locator_test.go#L20-L21) and [this](https://github.com/zucchinho/xk6-browser/blob/2aa3ad11b531c0613e39d8a6a70f3d7120605bb2/tests/locator_test.go#L182-L184) as a guide.

Reply:
Yes, the browsers will run in parallel.

Reply:
i more mean in terms of resources available to the CI agent, is it actually more performant to run a separate instance of a browser for each test, even if they are running in parallel? or just spin up a single instance of a browser and then sequentially run the tests against that instance? most of the test execution time will be the instantiation of the browser. i'm happy to make the change anyway

Reply:
oh looking closer at the examples you linked, i think i understand a bit better what you mean. we'd like the run the happy_path in parallel with all of the error cases, which are run sequentially?

Reply:
Yeah, you are right to raise concern whether there's any benefit to running them in parallel. At the moment it helps in a couple of ways:
- We think in general it does help speed the whole test suite.
- It isolates tests in their own browser instance. This prevents tests from interfering with each other due to any caching or config changes.

We parallelise all tests (including sub tests within a table test), unless there's a very good reason not to (at least for now).

Reply:
no problem, have done that